1xx-enable-ssl-for-kafka-bridge.md

```diff
@@ -0,0 +1,155 @@
+# Add support for TLS/SSL on the HTTP interface 
+
+This proposal recommends adding an option that can enable TLS encryption on HTTP interface so that requests from clients to the Kafka Bridge can be sent with encryption.
+
+## Current situation
+
+Strimzi does not support TLS/SSL for the HTTP Bridge, so client connections to the HTTP Bridge are unencrypted. A single listener serves all endpoints for Kafka operations and admin operations such as health checks and monitoring. The default port is 8080.
+
+## Motivation
+
+The connection between HTTP Bridge and Kafka cluster can be secured. However, there is currently no support for securing client connections to the HTTP Bridge. The current recommendation is to use firewalls or API gateways to secure the client connections. This requires users to make additional configurations and does not provide the best user experience. 
+
+Moreover, some of the endpoints are for internal use only, such as `/healthy`, `/ready` and `/metrics` but they are exposed on the same listener and the same port as the other endpoints that are used by external clients. This means the endpoints for internal use only are also exposed to external clients. And if SSL is enabled, it will introduce more operational complexity as these internal endpoints would need to be configured with TLS and certificates.
+
+## Proposal
+
+This proposal introduces adds new configurations to enable an SSL server and load a keystore certificate and key in PEM format so that client connections to HTTP Bridge can be encrypted. The new configurations will be added to the existing `http` prefixed configurations:
+- http.ssl.enable
+- http.ssl.keystore.certificate.location
+- http.ssl.keystore.key.location
+- http.ssl.keystore.certificate.chain
+- http.ssl.keystore.key
+- http.ssl.enabled.protocols
+- http.ssl.enabled.cipher.suites
+- http.management.port
+
+When `http.ssl.enable` is set to `true`, the HTTP Bridge server starts with SSL enabled and the new configurations will allow users to define locations of the keystore files or certificate chain and key in `PEM` format. Additional formats such as `PKCS12` and `JKS` might be supported in the future if users request them.
```

Comment:
So AFAIU the user should set `http.ssl.keystore.certificate.location` together with `http.ssl.keystore.key.location` OR `http.ssl.keystore.certificate.chain` together with `http.ssl.keystore.key`. Is my understanding right? Should we make it more explicit?